govcd/generic_functions.go

```diff
@@ -36,3 +42,34 @@ func oneOrError[T any](key, name string, entitySlice []*T) (*T, error) {
 
 	return entitySlice[0], nil
 }
+
+// readFileAndUnmarshalJSON reads a file and unmarshals it to the given variable
+func readFileAndUnmarshalJSON[T any](filename string, object T) error {
```

Comment:
Why using generics here? The object is `any`, as it the parameter to `json.Unmarshal`, so we don't gain anything by using a definable type

Reply:
True, the reason was just to extract the I/O functions so we could test them. I guess I can just make these into normal functions.

Reply:
I made the functions non-generic, but I don't know where I should move these utility functions, as they don't fit in generic_functions.go

Reply:
I think these functions can stay in api_token.go, as they are private functions that deal with such matter